modules/rtm/tests/quicktest.R

```diff
@@ -0,0 +1,20 @@
+## Quick test
```

Comment:
These look more like demonstrations than tests. More suitable as vignettes. For tests, please use the testthat testing framework.